### PR TITLE
Added the interesting gene list as optional

### PR DESCRIPTION
**Pull request by silviamorins**
now also the .txt file containing a list of the genes of interest can be provided to the report: if the argument is there, the corresponding session gets showed, otherwise it doesn't.

this is related to [this issue](https://github.com/qbicsoftware/rnadeseq/issues/10).

### CODE REVIEW

**Comment by silviamorins** [comment]
I also added a testdata/ folder containing the test data from the r-analysis-template repo

**Comment by silviamorins** [comment]
After this is merged, I will address the paths to `qbicsoftware/rnadeseq/dev`

File: .Rhistory [hunk 1]
```diff
@@ -0,0 +1,2 @@
+a()
+q()
```
Comment (ggabernet) [inline .Rhistory]: Please add .Rhistory to the .gitignore file so the changes there are not pushed

Comment (silviamorins) [reply .Rhistory]: thanks, done

**Pull request merged**